README.md

```diff
@@ -274,9 +274,8 @@ Refer to the [Contributing guidelines](CONTRIBUTING.md).
 
 ## <picture><source height="20" width="20" media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/ibm-telemetry/telemetry-js/main/docs/images/ibm-telemetry-dark.svg"><source height="20" width="20" media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/ibm-telemetry/telemetry-js/main/docs/images/ibm-telemetry-light.svg"><img height="20" width="20" alt="IBM Telemetry" src="https://raw.githubusercontent.com/ibm-telemetry/telemetry-js/main/docs/images/ibm-telemetry-light.svg"></picture> IBM Telemetry
 
-This package uses IBM Telemetry to collect anonymous metrics data in CI environments. By installing this
```

Comment:
Can we keep the language "in CI environments?" Is that still true?

Reply:
@metonym Hi, we are changing the telemetry notices in our instrumented repos to our latest recommended notice found [here](https://github.com/ibm-telemetry/telemetry-js?tab=readme-ov-file#5-add-telemetry-collection-notice-to-your-docs) to be more precise in stating how collected data is anonymized and de-identified. While we removed the specific wording of "in CI environments", nothing has changed in that regard as seen [here](https://github.com/ibm-telemetry/telemetry-js?tab=readme-ov-file#when-does-data-get-collected) in our README. 

That being said, we can keep "in CI environments" if you'd like to, so long as the anonymized data message remains in the notice. Let me know what you'd like to do!

Reply:
Done!